servicetalk-http-api/src/main/java/io/servicetalk/http/api/DefaultHttpExecutionStrategy.java

```diff
@@ -125,20 +126,22 @@ public StreamingHttpService offloadService(final Executor fallback, final Stream
                                                         StreamingHttpRequest request,
                                                         final StreamingHttpResponseFactory responseFactory) {
```

Comment:
`invokeService` should also be updated, or build this PR on top of  https://github.com/apple/servicetalk/pull/1710 and update the `OffloadFilter`.

Reply:
Since #1710 is nearing completion and #1723 is completed I will rebase this on top of #1710